render/gles2/renderer.c

```diff
@@ -14,6 +14,10 @@
 #include <wlr/util/log.h>
 #include "render/gles2.h"
 
+#ifndef GL_RENDERBUFFER_OES
+#define GL_RENDERBUFFER_OES 0x8D41
```

Comment:
We shouldn't do this

Reply:
You know where the header for this is?

Reply:
`<GLES2/gl2ext.h>`

Reply:
Should be in `GLES2/gl2ext.h`

Reply:
Odd, I don't have it in mine. Are you guys running arch too?

Reply:
Oh, in this case, `GL_RENDERBUFFER` is part of core GLES2. You just need to use that.

Reply:
Oh, I see. I guess that only applies the GL ES 1.0 then. I was reading this: https://www.khronos.org/registry/OpenGL/extensions/OES/OES_EGL_image.txt